src/utils/CrossChainBridge.ts

```diff
@@ -73,12 +75,34 @@ export const createAndReturnBridge = (bridgeAddress: Address, event: ethereum.Ev
 export const createAndReturnFederation = (federationAddress: Address, event: ethereum.Event): Federation => {
   let federation = Federation.load(federationAddress.toHex())
   if (federation == null) {
+    FederationTemplate.create(federationAddress)
+    log.info('Federation created: {}', [federationAddress.toHex()])
     federation = new Federation(federationAddress.toHex())
+    const federationContract = FederationContract.bind(federationAddress)
+
     federation.members = []
```

Comment:
I'm not sure you need to set federeation.members to an empty array if you are then re-setting in in line 92.

I think you can just have line 88 be ```const membersArray = []``, and then push the members into there and then line 92 will do the rest

Reply:
I will need to retest it, but I think I made it intentionally because otherwise subgraph didn't build.
try_getMembers() returns array of `Address` and `federations.members` wants `Bytes`, and somehow if line `88` wasn't set to federations.members but to empty array it failed

Reply:
Update: build works, but subgraph crashes when event is received and it tries to assign members. @BetsyBraddock

Reply:
What's the error when it crashes?